scripts/clean_osm_data.py

```diff
@@ -300,6 +300,51 @@ def split_cells_multiple(df, list_col=["cables", "voltage"]):
     return df  # return new frame
 
 
+# cable and circuit tags may be in a non-numeric format
+cables_tag_to_n_cables = {
+    "single": "1",
+    "triple": "3",
+    "partial": "1",
+    "1 disused": "0",
+    "3 disused": "0",
+    "1 (Looped - Haul & Return) + 1 power wire": "1",
+    "ground": "0",
+    "line": "1",
+    # assuming that in case of a typo there is at least one line
+    "`": "1",
+    "^1": "1",
+    "e": "1",
+    "d": "1",
+    "2-1": "3",
+    "3+3": "6",
+    "6+1": "6",
+    "2x3": "6",
+    "3x2": "6",
+    "2x2": "4",
+}
+
+circuits_tag_to_n_circuits = {
+    "1/3": "0",
+    "2/3": "0",
+    # assumption of two lines and one grounding wire
+    "2-1": "2",
+    "single": "1",
+    "partial": "1",
+    "1 disused": "0",
+    # assuming that in case of a typo there is at least one line
+    "`": "1",
+    "^1": "1",
+    "e": "1",
+    "d": "1",
+    "1.": "1",
+}
+
+# the circuit tags "1/3" and "2/3" are being dropped manually
+# according to checks of the corresponding power grids
+dropped_tags = [x for x in ["1/3", "2/3"] if circuits_tag_to_n_circuits[x] == "0"]
+print(dropped_tags)
```

Comment:
Maybe this print can be commented out.
May you explain better the line 344? Tomorrow we could discuss about it

Reply:
Yes, `print` was a part of testing and definitely need to be deleted. Thank you for noticing it.

The line 344 was aimed to select the dictionary keys to show a warning if they are dropped. Currently the tags `"1/3", "2/3"` are dropped with a warning while `"1 disused"` is being dropped silently. But probably it would be a better idea to show a warning always when anything is dropped...

Reply:
I'm ok with showing what codes are converted, but I'd say that it would be nice to show those that are dropped, according to the available data.
E.g. if we have a dataset in which "1/3" is not available, it is not dropped.
This would be maybe nicer and could be inserted somewhere around line 371 (for cables) and 404 (for circuits), but that is optional, if it fast to do on your side, we can incorporate it into this PR.

Generally, please use `_logger.info(....)` rather than `print`, this is required

Reply:
@davide-f, many thanks for the comments and the discussion! It was very helpful. I have added info messages and warnings and tried to make work with the dropped codes a little cleaner:

https://github.com/ekatef/pypsa-africa/blob/44f6f4429346abe590f585ec30436347d1715d2b/scripts/clean_osm_data.py#L342-L347